npm/metrics/promutil/util.go

```diff
@@ -0,0 +1,57 @@
+package promutil
```

Comment:
Can we move it out of promutil folder? or move its test file (test-util.go) in this folder. Usually test files remain next to the main files.

Reply:
test-util.go has utilities for testing. I'll move them into this file because this file is also utilities for testing

Reply:
If this is only for testing, then can you rename it test-util.go. does it need to be inside promutil?